sklearn/linear_model/tests/test_huber.py

```diff
@@ -0,0 +1,180 @@
+# Authors: Manoj Kumar mks542@nyu.edu
+# License: BSD 3 clause
+
+import numpy as np
+from scipy import optimize, sparse
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_greater
+
+from sklearn.datasets import make_regression
+from sklearn.linear_model import HuberRegressor, Ridge, SGDRegressor
+from sklearn.linear_model.huber import _huber_loss_and_gradient
+
+rng = np.random.RandomState(0)
+
+
+def make_regression_with_outliers(n_samples=50, n_features=20):
```

Comment:
Maybe add a one-line comment explaining the function. the data has outliers in X, not in y, right?